CRUDRecipeEF.PL/Bootstrap.cs

```diff
@@ -0,0 +1,71 @@
+﻿using CRUDRecipeEF.BL.DL.Data;
+using CRUDRecipeEF.BL.DL.Services;
+using CRUDRecipeEF.PL.Menus;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Serilog;
+using System;
+using System.IO;
+
+namespace CRUDRecipeEF.PL
+{
+    public static class Bootstrap
+    {
+        /// <summary>
+        /// Register services and app configs here
+        /// </summary>
+        /// <returns>IHostBuilder</returns>
+        public static IHostBuilder CreateHostBuilder(string[] args)
+        {
+            IConfigurationBuilder configBuilder = new ConfigurationBuilder()
+                .SetBasePath(Directory.GetCurrentDirectory())
+                .AddJsonFile("appsettings.json", optional: false, reloadOnChange: true)
```

Comment:
microsoft hosting adds the needed appsettings.json by default, is there any specific reason we need to do it again?